Integrations/CounterCraft/CounterCraft.md

```diff
@@ -0,0 +1,744 @@
+## Overview
+---
+
+CounterCraft Deception Solution detects advanced adversaries. Automate counterintelligence campaigns to discover targeted attacks with real-time active response.
+This integration was integrated and tested with version xx of CounterCraft Deception Director
+## CounterCraft Deception Director Playbook
+---
+
+## Use Cases
+---
+
+* Query IOCs (objects) in your Deception Director
+* Retrieve events from your deception campaigns
+* Retrieve configuration from your Deception Director
+* Retrieve alerts (notifications) from your Deception Director
+* Create new deception campaigns
+* Create new deception hosts
+* Operate your campaigns, hosts, services and breadcrumbs
+
+## Prerequisites
+---
+
+You need to obtain the following Deception Director information.
+
+* Server URL
+* API Key
+* Secret Key
+
+In order to obtain the API Key and the Secret Key you need to go to the user settings in the Deception Director
+and copy both or generate a new pair if they are not already generated.
```

Comment:
create new file called `CountetCraft_description.md` and copy there these lines.
description file should contain instructions on how to configure the instance in Demisto , where to generate API keys, secrets, url etc

Reply:
Ok!